ipywidgets/static/widgets/js/layout.js

```diff
@@ -36,18 +36,15 @@ define([
                 'align_items',
                 'align_self',
                 'bottom',
+                'border',
```

Comment:
Here's an idea, just use `border-width`, since it doesn't conflate the styling with the layout, and it itselft is a top-level shorthand: `https://developer.mozilla.org/en-US/docs/Web/CSS/border-width